src/chapters/setup-intro-data-types/setup.rst

```diff
@@ -196,7 +196,8 @@ Following the "Hello, World" trend, let's create a new IntelliJ project.
    The section on the left is the project's file tree. Clicking the triangle next to 
    the project name, ``HelloWorld``, displays the ``src`` file, followed by the base package
    we created, and finally our ``Main.java`` file. ``Main.java`` is also opened on the right
-   in this initial project view.
+   in this initial project view. See line 1, ``package com.company``? Java packages help
```

Comment:
The phrasing here is a little awkward. It seems like there are several separate ideas placed in the same paragraph.  Maybe shift "See line 1..." into a new paragraph?

Alternatively, maybe use something like, "In line 1, ``package com.company`` establishes a *package*, which Java uses to help us organize our code. It also..."